src/lib/Editor/RemoteResource/convertJSONAnnotationToInline.js

```diff
@@ -0,0 +1,25 @@
+export default async function convertJSONAnnotationToInline(json) {
+  try {
+    const response = await fetch(
+      'https://pubannotation.org/conversions/json2inline',
+      {
+        method: 'POST',
+        body: JSON.stringify(json),
+        headers: {
+          'Content-type': 'application/json'
+        }
+      }
+    )
+
+    if (!response.ok) {
+      response.text().then((errorMessage) => console.error(`${errorMessage}`))
```

Comment:
async関数なので次のように書けそうです。

```suggestion
      const errorMessaeg = await response.text()
      console.error(`${errorMessage}`)
```

Reply:
修正しました。
https://github.com/pubannotation/textae/pull/159/commits/f7e114c9fd663a8d22aaee6a3dbe194e9887d056#diff-c41f38ec8bc93cb5060c7d99c52b40adbcae1e0df37109b2f1d59e6040833dd6R13-R15